src/Components/AccountSummarySubscriptionForm/index.js

```diff
@@ -154,6 +154,47 @@ export class AccountSummarySubscriptionForm extends Component {
     }
   }
 
+  onUnsubscribeBtnHandler = async () => {
+    try {
+      const { unsubscribeUser } = this.props
+      const displayMsg = displayTexts.UNSUBSCRIPTION_SUCCESSFUL
+      await unsubscribeUser()
+      this.setState(
+        {
+          error: false,
+          displayMsg,
+          displaySubscriptionModal: false,
+        },
+        () =>
+          this.sendToast(1000, () => {
+            window.location.reload()
```

Comment:
Comment: I prefer to use a reload because there is a strange behaviour with the context